Makefile

```diff
@@ -147,6 +150,8 @@ kind-load-bundles: ## Load the e2e testdata container images into a kind cluster
 	$(CONTAINER_RUNTIME) build $(TESTDATA_DIR)/bundles/plain-v0/invalid-missing-crds -t testdata/bundles/plain-v0:invalid-missing-crds
 	$(CONTAINER_RUNTIME) build $(TESTDATA_DIR)/bundles/plain-v0/invalid-crds-and-crs -t testdata/bundles/plain-v0:invalid-crds-and-crs
 	$(CONTAINER_RUNTIME) build $(TESTDATA_DIR)/bundles/plain-v0/subdir -t testdata/bundles/plain-v0:subdir
+
+kind-load-bundles: build-bundles image-registry ## Load the e2e testdata container images into a kind cluster
```

Comment:
Maybe we don't want the image-registry step to be part of kind load bundles? We have two ways of surfacing image-bundles in the cluster for tests:
* `kind load` the image and have it directly available on the "node"
* push the image to the private docker registry and have it available for a Bundle resource to reference 

Maybe we should keep the existing flow for "public" test images and only utilize the registry for private image-related tests? so the `image-registry` and `registry-load-bundles` targets would only be part of the e2e setup, but separate from the existing `kind-load`.

Reply:
It makes sense, too.  I’ll change it. Thanks!